.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```

Comment:
Looks like the live server configurations were tracked during the Add stage? I'll update gitignore with the next pull request